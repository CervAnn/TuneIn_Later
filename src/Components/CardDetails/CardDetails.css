#cardDetails-container {
  display: flex;
  flex-flow: row wrap;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: #F4E697;
}

#album-name {
  font-size: 40px;
}

#artist-name {
  font-size: 30px;
}