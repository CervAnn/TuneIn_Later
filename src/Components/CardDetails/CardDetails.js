import React from 'react';
import { Link } from 'react-router-dom';
import './CardDetails.css';

const CardDetails = ({ artist_name, album_name, artwork_url, release_date,  primary_genre_name, returnRoute }) => {
  return (
    <div>
      <Link to={`${returnRoute}`} className='back-btn'>â—€ back</Link>
      <h1>{album_name}</h1>
      <h1>{artist_name}</h1>
      <img src={artwork_url} className='app-img-no-hover' />
      <p className='card-detail-genre'>{primary_genre_name}</p>
      <p className='card-detail-release_date'>{release_date}</p>
    </div>
  )
}

export default CardDetails;