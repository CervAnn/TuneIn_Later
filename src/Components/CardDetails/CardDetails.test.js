import React from 'react'
import { shallow } from 'enzyme'
import CardDetails from './CardDetails'

describe('CardDetails', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    let wrapper = shallow(<CardDetails 
      artist_name= "Beyonce"
      album_name="HOMECOMING"
      artwork_url="https://is2-ssl.mzstatic.com/image/thumb/Music/"
      release_date="2008-10-13T07:00:00Z"  
      primary_genre_name="Pop" 
      returnRoute="/"
    />)
    expect(wrapper).toMatchSnapshot()
  })
})