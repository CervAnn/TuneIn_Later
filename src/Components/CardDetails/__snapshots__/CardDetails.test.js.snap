// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDetails should match the snapshot with all data passed in correctly 1`] = `
<div>
  <Link
    className="back-btn"
    to="/"
  >
    â—€ back
  </Link>
  <h1>
    HOMECOMING
  </h1>
  <h1>
    Beyonce
  </h1>
  <img
    className="app-img-no-hover"
    src="https://is2-ssl.mzstatic.com/image/thumb/Music/"
  />
  <p
    className="card-detail-genre"
  >
    Pop
  </p>
  <p
    className="card-detail-release_date"
  >
    2008-10-13T07:00:00Z
  </p>
</div>
`;
